various-readers/src/main/java/com/dellemc/oe/readers/util/EarthQuakeRecord.java

```diff
@@ -0,0 +1,48 @@
+package com.dellemc.oe.readers.util;
+
+import org.apache.flink.table.api.TableSchema;
+import org.apache.flink.table.api.Types;
+import org.apache.flink.table.descriptors.Schema;
+
+public class EarthQuakeRecord {
+
+    public static TableSchema getTableSchema()
+    {
+        // Read data from the stream using Table reader
+        TableSchema tableSchema = TableSchema.builder()
+                .field("DateTime", Types.STRING())
```

Comment:
Use double and datetime data types where possible.

Reply:
Changed suitable DataTypes
I have a input date time string "2012/03/03 12:19:55.09"  But in Schema Types.SQL_TIMESTAMP() does not directly convert.
For now used String and added TODO comments to change in future.